packages/dd-trace/test/util.spec.js

```diff
@@ -39,4 +61,21 @@ describe('util', () => {
       expect(isFalse(String(v))).to.equal(false)
     })
   })
+
+  it('parseRules works', () => {
```

Comment:
This test could be split in smaller more descriptive tests.

Reply:
Ok! I will probably move the tests for this over to the config/prioritysampler tests then? Been a while since I wrote this and I agree the `parseRules` method is maybe not needed, and I can just `JSON.parse` in the `Config` class and `try/catch` in the `PrioritySampler`, so the detailed tests can maybe be split between the two respective test suites?

Reply:
I think it would make sense to move to priority sampler tests as well yes. Generally speaking all tests will soon be moved to integration tests, so the closer it is to actual usage from outside the library the better.